File: problems/problems_1833/solution.py
import solution


class Solution(solution.Solution):
    def solve(self, test_input=None):
        return self.maxIceCream(*test_input)

    def maxIceCream(self, costs, coins):
        """
        :type costs: List[int]
        :type coins: int
        :rtype: int
        """
        costs.sort(reverse=True)
        ans = 0
        while coins and costs:
            if coins < costs[-1]:
                return ans
            coins -= costs.pop()
            ans += 1
        return ans

File: problems/problems_1833/Solution.java
package problems.problems_1833;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public int maxIceCream(int[] costs, int coins) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        int[] costs = jsonArrayToIntArray(inputJsonValues[0]);
		int coins = Integer.parseInt(inputJsonValues[1]);
        return JSON.toJSON(maxIceCream(costs, coins));
    }
}
